### PR TITLE
Chore/re skin home screen

### PR DESCRIPTION
**Pull request by robbiejpfraser**
Not sure how much functionality you want to strip out of the home screen?

### CODE REVIEW

**Comment by samdods** [comment]
let's not merge this right away.

i think seeing the permissions if they're not setup fully is useful. imo we should only hide if they're all enabled, i.e. no further action required.

**Comment by robbiejpfraser** [comment]
> let's not merge this right away.
> 
> i think seeing the permissions if they're not setup fully is useful. imo we should only hide if they're all enabled, i.e. no further action required.

**Comment by robbiejpfraser** [comment]
Yeh probably worth making Anand aware of this to get some designs for these states

File: OpenTrace/Main/View Controllers/HomeViewController.swift [hunk 1]
```diff
@@ -76,58 +73,23 @@ class HomeViewController: UIViewController {
...
         }
     }
 
-    private func toggleIncompleteHeaderView() {
-        successHeaderView.isVisible = self.allPermissionOn
-        powerSaverCardView.isVisible = self.allPermissionOn
-        incompleteHeaderView.isVisible = !self.allPermissionOn
-        appPermissionsLabel.isVisible = !self.allPermissionOn
-    }
+	@IBAction func TrackMyConditionButtonTapped() {
```
Comment (samdods) [inline OpenTrace/Main/View Controllers/HomeViewController.swift]: shouldn't have capital T. but don't worry about it. i'll resolve conflicts and fix

**Pull request merged**